bugbug/models/annotate_ignore.py

```diff
@@ -71,7 +70,7 @@ def __init__(self, lemmatization: bool = False) -> None:
         )
 
         self.hyperparameter = {"n_jobs": utils.get_physical_cpu_count()}
-        self.clf = Pipeline(
+        self.clf = ImblearnPipeline(
```

Comment:
The `imblearn.pipeline.Pipeline` class inherits from `sklearn.pipeline.Pipeline`. The difference is that `ImblearnPipeline` supports `fit_resample`, so when fitting, if a transformer implements the `fit_resample` method, it will be used instead of `fit` or `fit_transform`.